### PR TITLE
`Paywalls`: polish template 3

### PR DESCRIPTION
**Pull request by NachoSoto**
image

### CODE REVIEW

File: ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt [hunk 1]
```diff
@@ -313,8 +314,8 @@ private fun AnnotatedString.Builder.appendMarkdownChildren(
                 append(child.literal)
                 pop()
             }
-            is HardLineBreak -> {
-                append("\n")
+            is HardLineBreak, is SoftLineBreak -> {
+                appendLine()
```
Comment (NachoSoto) [inline ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Markdown.kt:318]: This fixes line breaks from Markdown

File: ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt [hunk 2]
```diff
@@ -88,26 +92,28 @@ internal fun Template3(
 }
 
 @Composable
-private fun Template3MainContent(state: PaywallViewState.Loaded) {
+private fun ColumnScope.Template3MainContent(state: PaywallViewState.Loaded) {
     IconImage(
         uri = state.templateConfiguration.images.iconUri,
         maxWidth = Template3UIConstants.iconSize,
         iconCornerRadius = Template3UIConstants.iconCornerRadius,
+        childModifier = Modifier.padding(top = UIConstant.defaultVerticalSpacing),
```
Comment (vegaro) [inline ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt:100]: 👍

File: ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template3TestData.kt [hunk 3]
```diff
@@ -62,31 +62,10 @@ internal val TestData.template3: PaywallData
...
-                        content = "You'll automatically get subscribed. " +
-                            "Cancel anytime before if you didn't love our app.",
-                        iconID = "attachment",
-                    ),
-                    PaywallData.LocalizedConfiguration.Feature(
-                        title = "Day 14",
-                        content = "You'll automatically get subscribed. " +
-                            "Cancel anytime before if you didn't love our app.",
-                        iconID = "attachment",
-                    ),
```
Comment (vegaro) [inline ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template3TestData.kt:86]: Have you tested it with a bunch of features?

**Review by vegaro — APPROVED** [review]

File: ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template3TestData.kt (hunk 3 above)
Comment (NachoSoto) [reply ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/testdata/templates/Template3TestData.kt:86]: Yeah it still works well. I just wanted this to look the same as the mock ups / iOS so we can compare the most common use case of features being vertically centered 😇

**Pull request merged**